week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/shop/hero.component.ts

```diff
@@ -0,0 +1,20 @@
+import { div, p, section } from '../framework/dom-creators.js';
+
+interface IHeroProps {
+  title: string;
+  subTitle: string;
+}
```

Comment:
Nie jestem fanem przedrostka `I` moim zdaniem - zbędny zabieg. Jednak to już kwestia "gustu" czy "coding standards" :) 

Natomiast fajnie że masz to lokalnie rozdzielone jako props i nie exportujesz - bo nie ma takiej potrzeby 👍